moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js

```diff
@@ -114,7 +114,7 @@ const ProjectSummaryProjectDescription = ({
         )}
         {!editMode && (
           <ProjectSummaryLabel
-            text={description || " - "}
+            text={description.trim().length > 0 ? description : " - "}
```

Comment:
This is to fix a bug I found where if you save empty spaces as a project description, you can't edit the description again (ex: https://moped.austinmobility.io/moped/projects/1790) `Boolean("  ")` is true.

Reply:
thank you for fixing this!

we have [a snackoo](https://github.com/cityofaustin/atd-data-tech/issues/10339) to prevent users from saving an empty description in the first place. we require description on project create but not project edit.